src/ui/profile_preferences.ui

```diff
@@ -769,6 +769,16 @@
             </property>
            </widget>
           </item>
+          <item row="2" column="0">
+           <widget class="QCheckBox" name="checkBox_echoLuaErrors">
+            <property name="toolTip">
+             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Prints Lua errors to the main console.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
How about "Prints Lua errors to the main window as well as the errors view." so have a bit more clarity